tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -550,15 +552,16 @@ func TestRateLimit(t *testing.T) {
 	got, err := vectorValue(value, map[string]string{responseCodeLabel: "429", "destination_version": "v1"})
 	if err != nil {
 		t.Logf("prometheus values for request_count:\n%s", promDump(promAPI, "request_count"))
-		fatalf(t, "Could not find rate limit value: %v", err)
+		errorf(t, "Could not find 429s: %v", err)
+		got = 0 // want to see 200 rate even if a bug makes there is no 429s
 	}
 
 	// establish some baseline to protect against flakiness due to randomness in routing
```

Comment:
Is this comment still valid? Does the test still use v2-v3 routing rule?

Reply:
it's not I think - tbh I think the code could be simpler and check for closer to exact 1rps 200s, rest as 429s but my goal in this PR is to show actual rates when things fail